### DISCUSSION TITLE
https://github.com/microsoft/chat-copilot/ has been removed from the semantic kernel documentation

### DISCUSSION THREAD

**Discussion by christianarg** [microsoft/semantic-kernel#9209] [General]
Hi,
I've realized https://github.com/microsoft/chat-copilot/ has been removed from the semantic kernel documentation. I think this happened some months ago when you refactored the docs. Was this intended? If yes, could you share why? It was a nice reference app.

**Comment by evchaki** [comment]
@christianarg - thanks for asking.  Chat Copilot was sample code that we released 18 months ago to get the community ready to build their own copilots.  Over that time, Microsoft has released many samples and tools to help you do this.  We did remove it from the learn site and made sure we posted a message on the top of the repo letting people know it is only a sample and not to use it in production.

**Reply by christianarg** [reply]
yes I totally understand that it's a sample and shouldn't be used in production. Can you provide links to those newest Microsoft samples that Microsoft released as sample/reference applications please?

**Reply by evchaki** [reply]
Here are a couple to look at:
- eShop Support - https://github.com/dotnet/eShopSupport and https://github.com/Azure-Samples/eShopOnAzure
- Project Miyagi - https://github.com/Azure-Samples/miyagi
- GenAI in a Box - https://github.com/Azure-Samples/gen-ai-bot-in-a-box
